src/group/libp2p_group_mgr.erl

```diff
@@ -86,15 +93,25 @@ handle_call({remove_group, GroupID}, _From, #state{tid = TID} = State) ->
             lager:warning("removing missing group ~p", [GroupID])
     end,
     {reply, ok, State};
+handle_call(force_gc, _From, #state{group_deletion_predicate = Predicate,
+                                    storage_dir = Dir} = State) ->
+    lager:info("forcing gc"),
+    Reply = gc(Predicate, Dir),
+    {reply, Reply, State};
 handle_call(_Request, _From, State) ->
     lager:warning("unexpected call ~p from ~p", [_Request, _From]),
-    Reply = ok,
-    {reply, Reply, State}.
+    {noreply, State}.
```

Comment:
Won't this deadlock the caller (or trigger a timeout)?

Reply:
yeah.  I do this to call out bad calls by forcing them to error, hopefully in test.